mkosi.default.d/debian/10-mkosi.debian

```diff
@@ -73,6 +73,9 @@ Packages=
         locales
         nano
         strace
+        xxd
+        libasan8
+        libubsan1
```

Comment:
On Debian the `gcc` package depends on `libasan*` and `libubsan*` transitively via `gcc-*` and `libgcc-*-dev` so they are pulled automatically when `gcc` is installed. I think it would be better to leave out these packages and let `gcc` pull the right dependencies to avoid editing this file every time `gcc` gets updated.

Also, it seems `xxd` is commented out everywhere else.

Reply:
The gcc package is not installed in the final image, only in the build image so we need to pull in the libasan/libubsan deps explicitly there.

xxd isn't commented out, it's a comment explaining what the package below that comment is providing, will clarify the comments

Reply:
> we need to pull in the libasan/libubsan deps explicitly there

I wonder why `gcc` can't be installed instead? `libasan8` and `libubsan1` are going to get out of sync with `gcc` used to build `systemd`. If it can't be installed here for some reason I think `gcc-11` should be installed explicitly to catch the mismatch when `systemd` is built.